www/gatsby-config.js

```diff
@@ -34,6 +34,7 @@ const plugins = [
     },
   },
   'gatsby-plugin-react-helmet',
+  'gatsby-plugin-typescript',
```

Comment:
Note for reviewers: you may have noticed there are no `package.json` changes adding this, that is because `gatsby-plugin-typescript` is already a dependency of `gatsby` (v4 at least - I assume later versions as well) https://www.npmjs.com/package/gatsby/v/4.14.1?activeTab=dependencies